DashAI/back/dataloaders/classes/dataset_dashai.py

```diff
@@ -0,0 +1,151 @@
+import json
+import os
+from typing import Dict, List
+
+from datasets import ClassLabel, Dataset, Value, load_from_disk
+
+
+class DatasetDashAI(Dataset):
+    def __init__(
+        self,
+        table,
+        inputs_columns: List[str],
+        outputs_columns: List[str],
+        *args,
+        **kwargs,
+    ):
+        """
+        Dataset from Hugging Face with more metadata about dataset.
+        Args:
+            table: Table
+            inputs_columns: list
+            outputs_columns: list
+            *args: arguments forwarded to super.
+            **kwargs: keyword arguments forwarded to super.
+        """
+        super().__init__(table, *args, **kwargs)
+        self.validate_inputs_outputs(self.column_names, inputs_columns, outputs_columns)
+        self._inputs_columns = inputs_columns
+        self._outputs_columns = outputs_columns
+
+    @property
+    def inputs_columns(self):
+        return self._inputs_columns
+
+    @inputs_columns.setter
+    def inputs_columns(self, columns_names: List[str]):
+        self._inputs_columns = columns_names
+
+    @property
+    def outputs_columns(self):
+        return self._outputs_columns
+
+    @outputs_columns.setter
+    def outputs_columns(self, columns_names: List[str]):
+        self._outputs_columns = columns_names
+
+    @staticmethod
+    def validate_inputs_outputs(
+        names: List[str], inputs: List[str], outputs: List[str]
+    ):
+        """
+        Validates the columns to be chosen as input and output considering those that
+        already exist in the dataset.
+        Args:
+            names (list): dataset column names
+            inputs (list): list of names to be input columns
+            outputs(list): list of names to be output columns
+        """
```

Comment:
Numpy convention.
Also note the first line should fit in the max line length (89).
If your description is too long, split it and put the secondary info after the short initial description.

```suggestion
        """Validate the columns to be chosen as input and output.
        
        The algorithm considers those that already exist in the dataset.

        Parameters
        ----------
        names : List[str]
            dataset column names
        inputs : List[str]
            list of names to be input columns
        outputs : List[str]
            list of names to be output columns

        """
```